topomodelx/nn/hypergraph/allset.py

```diff
@@ -16,18 +16,16 @@ class AllSet(torch.nn.Module):
         Dimension of the input features.
     hidden_channels : int
         Dimension of the hidden features.
-    out_channels : int
-        Dimension of the output features.
     n_layers : int, default: 2
         Number of AllSet layers in the network.
-    dropout : float
-        Dropout probability.
+    layer_dropout:
+        Dropout probability for the AllSet layer.
     mlp_num_layers : int, default: 2
         Number of layers in the MLP.
-    mlp_activation : torch.nn.Module, default: None
-        Activation function for the MLP.
     mlp_dropout : float, default: 0.0
         Dropout probability for the MLP.
```

Comment:
missing parameter type: e.g. `dropout : float` and note the space before : according to numpy's conventions for docstring, which we use here.